test/apitest.py

```diff
@@ -862,7 +862,7 @@ def test_patch_routes_drop_bm_routes(self):
         routes_not_bm = []
         for route in routes:
              del route['cmd']
-             if route['ip_prefix'] == '10.1.1.1/32' or  route['ip_prefix'] == '10.1.5.1/32':
+             if route['nexthop'] == '34.53.1.0':
```

Comment:
A possible improvement could be adding a unit test for genVxlanTunnelInfo(loading vxlan tunnel data for pre-configured tunnel). eg: test_guid_persistence tests for vnet in a similar type of scenario.

Reply:
Agree it can be a revised testing, but can I do it as a future PR? or do you prefer to handle it in this PR itself?

Reply:
Approved the changes, future PR works from my perspective.